webserver/templates/data/summary.html

```diff
@@ -28,12 +28,16 @@ <h3 class="text-center">
     <table class="table table-striped table-bordered">
       <thead><tr><th style="width: 30%;">Tonal &amp; Rhythm</th><th>value</th></tr></thead>
       <tr>
-        <td><a href="http://essentia.upf.edu/documentation/reference/streaming_ChordsDescriptors.html">key</a></td>
-        <td>{{ data.lowlevel.tonal.key_key }}</td>
+        <td><a href="http://essentia.upf.edu/documentation/reference/streaming_Key.html">key</a></td>
+        <td>{{ data.lowlevel.tonal.key_key }} {{ data.lowlevel.tonal.key_scale }}
+            {% if data.lowlevel.tonal.key_strength %}
+            ({{ data.lowlevel.tonal.key_strength|round(2)*100 }}%)
```

Comment:
How do you combine indentation of tags and statements?  would you consider this:

```
<td>{{ data.lowlevel.tonal.key_key }} {{ data.lowlevel.tonal.key_scale }}
    {% if data.lowlevel.tonal.key_strength %}
        ({{ data.lowlevel.tonal.key_strength|round(2)*100 }}%)
    {% endif %}
</td>
```
?